confmap/provider/eecprovider/README.md

```diff
@@ -0,0 +1,28 @@
+# EEC Provider
+
+This is an OpenTelemetry Collector `confmap.Provider` module that allows the
+Collector to be configured with the Dynatrace Extensions Execution Controller.
+
+> [!WARNING]
+> This is not intended for direct customer use, but is only intended for use by
+> Collectors installed and managed by Dynatrace. Configuring this
+> component directly is not supported.
+
+## Configuration
+
+This confmap Provider can be configured by pasing query parameter-formatted values
+inside the fragment of the URL given to the config flag. For example:
+
+```text
```

Comment:
```suggestion
```shell
```

Reply:
I tried this, but the problem I ran into is that the config fragments are highlighted as comments, at least in my local Markdown renderer. I'll give it a shot with the GitHub renderer and see if it's any different.

Reply:
You can see what it looks like here: https://github.com/Dynatrace/dynatrace-otel-collector/commit/eec2b66eb3dd4d3c8bc3b62853a88decebed54ad?short_path=5b27679#diff-5b27679011dbb54249c6f87e02849959e30682d84058764b1a8bedfd3957c6c7

I think the highlighted ampersand is more distracting than anything, I'd like to opt to leave it as plain text for now. Let me know if you think the highlighting would still be worth it.

Reply:
it's just a nitpick. I leave the decision on you.